admin/templates/project/streamlit.yaml

```diff
@@ -0,0 +1,82 @@
+#!jinja2: lstrip_blocks: True
+{#
+    REQUIRED APPLICATION VARIABLES
+    ------------------------------
+    image : Docker image path
+    port  : Container port which public HTTP service runs on
+    replicas: how many instances of the application to run
+#}
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: streamlit-app
+  namespace: {{ inventory_hostname }}
+  labels:
+    app: {{ inventory_hostname }}
+spec:
+  replicas: {{ app.replicas }}
+  strategy:
+    type: Recreate
```

Comment:
I followed another template in using this strategy, but [other strategies](https://github.com/ContainerSolutions/k8s-deployment-strategies) are often preferable: shadow, blue/green, or canary...